### PR TITLE
Change requirements so that they install on Mac

### PR DESCRIPTION
**Pull request by maciek-pioro**
# Description
With this change I can install install requirements on MacOS. It shouldn't change an awful lot on other machines. I think we should merge if the tests pass without creating new images.

# Checklist
   - [x] Is this PR focused on a single feature without the possibility of being divided into smaller PRs?
   - [x] Are all variables set with logical defaults that are backward compatible?
   - [x] I attached link to neptune if it was necessary
   - [x] Have you successfully executed the linter and tests?
   - [x] Are all functions concise and don't require splitting?
   - [x] Is there documentation for arguments and complex logic?
   - [x] Are there no temporary solutions in the code? Should a task be added to Trello to refine the code?
   - [x] Are all functions placed in appropriate files?
   - [x] Is the PR name meaningful and descriptive?
   - [x] Is PR description provided, or unnecessary?
  
Other questions:
   - [ ] ~If I made changes to the requirements or anything related to the Singularity image I did generate new image and upload a new image to the clusters~ I don't think we need that
   - [ ] ~If this change is significant enough I notified all individuals working with this repository (@channel in slack channel)~ Not significant
   - [ ] ~If there is a need for a second reviewer I requested additional review~ No need

# Reviewer's checklist:
   - [ ] Does the Neptune link work and does it actually compare the situation before and after the PR?
   - [ ] Do I understand the code?
   - [ ] Do I think another reviewer is not needed or I commented that this PR need another reviewer?

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -6,7 +6,7 @@ matplotlib==3.5.1
 opt-einsum==3.3.0
 line-profiler==4.0.3
 einops==0.4.0
-tensorflow==2.8.0
+tensorflow>=2.8.0
```
Comment (Simontwice) [inline requirements.txt:9]: this is tough, what's the reason for dropping the exact version?

**Comment by Simontwice** [comment]
has this been tested? Have you made a new singularity image that satisfy these requirements? does it work?

File: requirements.txt (hunk 1 above)
Comment (maciek-pioro) [reply requirements.txt]: Fixed.

**Comment by crewtool** [comment]
tensorflow 2.13 dodaje mnóstwo takich logów:
`2023-11-13 22:24:38.935759: W tensorflow/compiler/tf2tensorrt/utils/py_utils.cc:38] TF-TRT Warning: Could not find TensorRT` 
I issue które to naprawia, sugeruje instalację tensorflow 2.8...
https://github.com/tensorflow/tensorflow/issues/61468

**Comment by maciek-pioro** [comment]
@Simontwice @michalkrutul 
dodałem tak, że na macu jest TF 2.13, a na pozostałych systemach pozostaje po staremu. Ok?

**Review by crewtool — APPROVED** [review]
Lgtm

**Comment by maciek-pioro** [comment]
> Lgtm

**Comment by maciek-pioro** [comment]
> @Simontwice @michalkrutul dodałem tak, że na macu jest TF 2.13, a na pozostałych systemach pozostaje po staremu. Ok?

@Simontwice ?

**Review by Simontwice — APPROVED** [review]

**Pull request merged**